test cases/common/125 shared module/module.c

```diff
@@ -9,54 +9,6 @@
   #endif
 #endif
 
-#ifdef _WIN32
```

Comment:
Ok, thanks for clarifying that.  I've added the necessary extra help for Cygwin.

I guess I have a concern this is the only example of shared_module in the test cases, and could be interpreted to be saying "don't ever use link_with:' with a shared_module, which just causes portability problems, but that's probably better addressed in documentation and as a separate issue.